stdlib/sysconfig.pyi

```diff
@@ -30,6 +30,7 @@ def get_scheme_names() -> tuple[str, ...]: ...
 if sys.version_info >= (3, 10):
     def get_default_scheme() -> str: ...
     def get_preferred_scheme(key: Literal["prefix", "home", "user"]) -> str: ...
+    def _get_preferred_schemes() -> dict[Literal["prefix", "home", "user"], str]: ...
```

Comment:
Since we usually don't include underscore-prefixed symbols in stubs:

```suggestion
    # Documented -- see https://docs.python.org/3/library/sysconfig.html#sysconfig._get_preferred_schemes
    def _get_preferred_schemes() -> dict[Literal["prefix", "home", "user"], str]: ...
```